app/plugins/toa-logger.js

```diff
@@ -25,12 +25,18 @@ module.exports = {
             this.temp[wizardID] = {};
           }
 
+          if (command === 'BattleTrialTowerStart_v2') {
+            this.log_trial_boss(proxy, req, resp);
```

Comment:
Maybe it would be a good idea to use the `this.temp[wizardID] = {}` object to temprarely save data across different events. I think we do this in the run logger plugin too. So no extra temp file needed.